.github/workflows/CI2.yml

```diff
@@ -166,14 +205,33 @@ jobs:
             -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}
           cmake --build build --parallel 2
           cmake --install build
+      - name: Checkout heffte
+        if: ${{ matrix.heffte != 'NoheFFTe' }}
+        # actions/checkout doesn't work for external repos yet
+        run: |
+          git clone --depth 1 --branch v2.0.0 https://bitbucket.org/icl/heffte.git heffte
+      - name: Build heffte
+        if: ${{ matrix.heffte != 'NoheFFTe' }}
+        working-directory: heffte
+        run: |
+          cmake -B build \
+            -DCMAKE_CXX_STANDARD="11" \
+            -DBUILD_SHARED_LIBS=ON \
+            -DHeffte_ENABLE_${{ matrix.heffte }}=ON \
```

Comment:
Uh, that is ugly.

Reply:
I don't love it. Would you prefer to build the list of optional flags like we used to with travis?

Reply:
we could do the same in shell above
```
[[ ${{ matrix.heffte }} = "heFFTe_with_FFTW" ]] && cmake_opts+=(  -DHeffte_ENABLE_FFTW=ON )
[[ ${{ matrix.heffte }} = "heFFTe_with_MKL" ]] && cmake_opts+=(  -DHeffte_ENABLE_MKL=ON )
```